js/ai/src/generate.ts

```diff
@@ -363,23 +367,36 @@ export class GenerateResponseChunk<T = unknown>
 export async function toGenerateRequest(
```

Comment:
Can you support raw string as input to `generate` as well?

```ts
ai.generate("tell me a long story");
```

(if it's already in here sorry, i missed it)

Reply:
yes, it's already supported:
https://github.com/firebase/genkit/pull/1019/files#diff-b79c0173e57ae8b5085835ee92582688df1f89dc5e223875cbbc4399d7e790f8R42